apps/climatemappedafrica/src/components/HURUmap/ChartFilter/index.js

```diff
@@ -86,44 +83,97 @@ function ChartFilter({
   }
 
   return (
-    <div className={classes.root}>
+    <Box
+      sx={({ typography }) => ({
+        paddingBottom: typography.pxToRem(10),
+      })}
+    >
       <Grid container alignItems="flex-end">
         {attributeOptions?.length > 0 && (
-          <Grid item className={classes.grid}>
+          <Grid
+            sx={({ typography }) => ({
+              marginRight: { md: typography.pxToRem(10) },
+            })}
+          >
             <Select
               helperText={attributeText}
               options={attributeOptions}
               selected={selectedAttribute}
               onChange={onAtrributeChange}
               disabled={index === "default"}
-              classes={{
-                select: classes.select,
-                filled: classes.filled,
-                paper: classes.selectPaper,
+              SelectProps={{
+                width: theme.typography.pxToRem(205),
+                height: theme.typography.pxToRem(40),
+                paddingLeft: 0,
+                "& .MuiSelect-filled": {
+                  padding: `${theme.typography.pxToRem(10)} 0 ${theme.typography.pxToRem(
+                    10,
+                  )} ${theme.typography.pxToRem(15)} !important`,
+                },
+              }}
+              MenuProps={{
+                "& .MuiMenu-paper": {
+                  height: theme.typography.pxToRem(310),
+                  boxShadow: "none",
+                },
+                "& .MuiMenu-list": {
+                  padding: `${theme.typography.pxToRem(10)} 0 ${theme.typography.pxToRem(
+                    10,
+                  )} ${theme.typography.pxToRem(15)} !important`,
```

Comment:
I think this one should be set to 0, because if we set any value it would look a bit awkward with some extra padding on the left
<img width="866" alt="Screenshot 2024-12-04 at 14 09 10" src="https://github.com/user-attachments/assets/49029368-d568-4ed0-89d4-e8055831c1ca">